src/audio-worklet/migration/worklet-recorder/app.js

```diff
@@ -327,7 +327,7 @@ const createRecord = (recordingProperties, recordingLength, sampleRate,
   const recordingUrl = createLinkFromAudioBuffer(recordingBuffer, true);
 
   player.src = recordingUrl;
-  downloadLink.src = recordingUrl;
+  downloadLink.href = recordingUrl;
```

Comment:
Hah! That's a nice catch. Do you have a staging page link from your fork?

Reply:
No, I didn't. And I am wondering why "Build and deploy to GitHub Pages / Build and deploy to GitHub Pages (pull_request)" It seems like I can not pull request to the main branch due to the deploy key?

Reply:
If the auto deploy is not possible, could you locally check if the patch fixes the issue? (I believe you already did, but just to double check)

Reply:
Yes, the issue is fixed